README.md

```diff
@@ -66,7 +67,10 @@ Web Applications
   * [ ] if default-src is not `none`, frame-src, and object-src should be `none` or only allow specific origins
   * [ ] no use of unsafe-inline or unsafe-eval in script-src, style-src, and img-src
 * [ ] Third-party javascript must be pinned to specific versions using [Subresource Integrity (SRI)](https://infosec.mozilla.org/guidelines/web_security#subresource-integrity)
-* [ ] Web APIs must set a non-HTML content-type on all responses, including 300s, 400s and 500s
+* [ ] Web APIs:
+  * [ ] must set a non-HTML content-type on all responses, including 300s, 400s and 500s
+  * [ ] Web APIs should export an OpenAPI (Swagger) to facilitate automated vulnerability tests
+  * [ ] Should use authentication tokens with a unique pattern which is easily parsed with a regexp. This will allow easy inclusion into a token scanning service if needed in the future.
```

Comment:
nit: clarify (e.g. "using a unique prefix including the application name") and maybe provide an example

Reply:
good idea - added